package.json

```diff
@@ -36,7 +36,9 @@
     "test:browser": "sh ./scripts/test:browser.sh",
     "coverage": "cat ./coverage/lcov/lcov.info | codacy-coverage",
     "test": "npm run test:node && npm run test:browser",
-    "doc": "typedoc --out docs ./src"
+    "doc": "typedoc --out docs ./src",
+    "composeDown": "docker-compose -f ./test-dcore-node/docker-compose.yml down",
```

Comment:
Change also test:node.sh and test:browser.sh to use this.

Reply:
pls follow `package.json` nameing convetion "dcore:compose:up", "dcore:compose:down", etc.